notifications_utils/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '43.0.2'
+__version__ = '43.0.4'
```

Comment:
Assuming https://github.com/cds-snc/notification-utils/pull/61, version 43.0.3 will be merged before